src/routes/TrackedRoute.js

```diff
@@ -15,27 +16,40 @@ const TrackedRoute = ({
   disableViewEvent = false,
   render,
   ...rest
-}) => (
-  <Amplitude
-    eventProperties={inheritedProps => ({
-      ...inheritedProps,
-      scope: [...(inheritedProps.scope || []), ...scope],
-      study: rest.computedMatch && rest.computedMatch.params.kfId,
-      path: rest.location.pathname,
-    })}
-  >
-    {({logEvent}) => (
-      <>
-        {render ? (
-          <Route render={render} {...rest} />
-        ) : (
-          <Route component={component} {...rest} />
-        )}
-        {!disableViewEvent && logEvent('page view')}
-      </>
-    )}
-  </Amplitude>
-);
+}) => {
+  const eventProps = inheritedProps => {
+    if (rest.location.search) {
```

Comment:
You could probably save yourself from needing to repeat the full object within the `if` by just adding it to an existing object conditionally:
```
var eventProps =  { ...inheritedProps, ... };

if (rest.location.search) eventProps = { ...eventProps, utm_source: ... };
```